Cargo.toml

```diff
@@ -21,17 +21,18 @@ license = "MIT OR Apache-2.0"
 include = ["/src", "/tests", "LICENSE*"]
 
 [dependencies]
-bevy_replicon = { version = "0.26" }
-bevy_renet = { version = "0.0.11", default-features = false }
-bevy = { version = "0.13", default-features = false }
+bevy_replicon = { version = "0.27" }
+bevy_renet = { git = "https://github.com/raffaeleragni/renet", rev = "f36762ca7cf338300159be6206c191928a019f41", default-features = false }
```

Comment:
This should be updated once https://github.com/lucaspoffo/renet/pull/161 is merged.